backend/cli.py

```diff
@@ -151,22 +151,24 @@ async def providers_to_cache():
 
 
 @app.command()
-def remove_and_blacklist(media_id: str):
-    try:
-        db = database.SessionLocal()
-        media = get_media_by_id(db=db, media_id=media_id)
-        if not media:
-            typer.echo(f"Cannot find media: {media_id}")
-            raise typer.Abort()
+def remove_media(media_id: str, blacklist: bool = True):
+    db = database.SessionLocal()
```

Comment:
We should probably add a docstring with what it does - `remove-media` doesn't really say much